pack/return/rttcu_encounter.json

```diff
@@ -20,5 +539,569 @@
+        "code": "54056",
+        "double_sided": true,
+        "encounter_code": "return_to_before_the_black_throne",
+        "encounter_position": 1,
+        "faction_code": "mythos",
+        "name": "Return to Before the Black Throne",
+        "pack_code": "rttcu",
+        "position": 56,
+        "quantity": 1,
+        "text": "Perform the setup as indicated in the <i>The Circle Undone</i> Campaign Guide, with the following exceptions:\n- When gathering encounter sets, also gather the new encounter sets for <i>Return to Before the Black Throne</i> shown here.\n<b>(Continued on reverse side.)</b>",
+        "type_code": "scenario"
+    },
+    {
+        "back_flavor": "You stand on a precipice before the Black Throne and gaze up at the blind, voiceless monstrosity that is Azathoth. The mere sight of it causes your vision to blur and your body to shake uncontrollably. The monotonous piping is so loud now, it rattles your bones and causes blood to drip fro your eyes and ears. Should Azathoth awaken, it will devour every world that ever was.",
+        "back_name": "The End",
+        "back_text": "Check the Campaign Log.\n- If at least two of the following are true, you may proceed to <b>(→R2)</b>:<i>You accepted your fate</i>, <i>Mesmerizing Flute</i> is listed under\"Mementos Discovered\", and <i>Ritual Components</i> is listed under \"Mementos Discovered\"\n-If a least two if the following are true, you map proceed to <b>(→R3)</b>: <i>You rejected your fate</i>, <i>Scrap of Torn Shadow</i> is listed under \"Mementos Discovered,\" and <i>Wisp of Spectral Mist</i> is listed under \"Mementos Discovered\"\n- If <i>Strange Incantation</i> and <i>Bloody Tree Carvings</i> are listed under \"Mementos Discovered\", you pay proceed to <b>(→R6)</b>\n- Otherwise, <b>(→R4)</b>",
+        "clues": null,
+        "code": "54057",
+        "double_sided": true,
+        "encounter_code": "return_to_before_the_black_throne",
+        "encounter_position": 2,
+        "faction_code": "mythos",
+        "flavor" : "You are close now. You can feel the churning and twisting of aeons in the emptiness around you. Here, in the nucleus of all things - the very center of chaos itself - your objective, and your ultimate fate, lie at hand.",
+        "illustrator": "Reiko Murakami",
+        "name": "What Must Be Done (v. II)",
+        "pack_code": "rttcu",
+        "position": 57,
+        "quantity": 1,
+        "stage": 3,
+        "text": "[action] Spend X clues: Draw the top X cards of the Cosmos. Choose one to put into play using its <b>\"Cosmos -\"</b> instructions, and move to it. Shuffle the rest back into the Cosmos.\n<b>Objective</b> - If the lead investigator is a The Black Throne and there are no clues on ti, advance.",
```

Comment:
Please move the bold tags so that they are around Cosmos but nothing else:

`\"<b>Cosmos</b> - \"`

Reply:
Two typos at the end: 'at' the Black Throne, and 'it' not 'ti.

Reply:
Fixed the first one but I'm not sure there's a space between the - and "
``\"<b>Cosmos</b> - \" instructions``

I can add the space if it needs it.